### PR TITLE
FIX: `Evoked.decimate` is not updating `.first` and `.last`

### PR DESCRIPTION
**Pull request by wmvanvliet**
`Evoked.decimate()` fails to update the `.first` and `.last` attributes. I stumbled across the error when trying to save and then load a decimated `Evoked` object. This PR fixes it and extends the unit test to check these things.

### CODE REVIEW

**Comment by larsoner** [comment]
Travis is not happy

File: mne/tests/test_evoked.py [hunk 1]
```diff
@@ -34,22 +34,32 @@
...
     decim = dec_1 * dec_2
-    sfreq = 1000.
+    sfreq = 10.
     sfreq_new = sfreq / decim
-    data = rng.randn(n_epochs, n_channels, n_times)
-    events = np.array([np.arange(n_epochs), [0] * n_epochs, [1] * n_epochs]).T
+    data = rng.randn(n_channels, n_times)
     info = create_info(n_channels, sfreq, 'eeg')
     info['lowpass'] = sfreq_new / float(decim)
-    epochs = EpochsArray(data, info, events)
```
Comment (agramfort) [inline mne/tests/test_evoked.py]: why did you remove this test?

**Comment by agramfort** [comment]
can you add a what's new entry?

File: mne/tests/test_evoked.py (hunk 1 above)
Comment (wmvanvliet) [reply mne/tests/test_evoked.py]: Because it was testing the decimation of epochs, not evoked. I thought it was a copy/paste error

Comment (agramfort) [reply mne/tests/test_evoked.py]: did you check if it's tested elsewhere?

Comment (wmvanvliet) [reply mne/tests/test_evoked.py]: yes, an exact copy of these tests exists in `test_epochs.py`.

**Comment by agramfort** [comment]
thx @wmvanvliet
